.travis.yml

```diff
@@ -27,3 +27,10 @@ deploy:
     app: garage-it-smart-house-api-qa
     on:
       repo: garage-it/SmartHouse-backend
+  - provider: npm
+    email: paur.link@gmail.com
+    api_key:
```

Comment:
Can we save api_key to environment variable like this:
api_key: ${HEROKU_API_KEY} ?